plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/SqlJobResult.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.sql;
+
+public enum SqlJobResult {
+    NOT_RUN,
+    SUCCESS,
+    PARTIAL_SUCCESS,
+    FAILURE,
```

Comment:
Just `SUCCESS` and `PARTIAL_SUCCESS` are used. Do we really need to differentiate between these four states? Maybe it can be reduced down to just two: either success or not?

Reply:
Which status should we use if a script was partially executed and then encountered an error?

Reply:
But you don't handle these. Just the ones mentioned are used.

Reply:
You’re right! I haven’t come up with better names to reduce it to just two options yet, and I prefer not to use the SUCCESS status for cases where errors occur. I’ve removed `NOT_RUN` to streamline things a bit.

Reply:
And still, the difference between `SUCCESS` and `PARTIAL_SUCCESS` is not clear - they're both handled the same way. It seems that you want to use a `boolean` flag to indicate whether the execution was successful or not.

Reply:
Yes, they are handled the same way, but it makes the code more readable. I’m open to suggestions for naming that boolean to maintain a consistent level of abstraction between SQLQueryJob and SQLQueryListener.